### PR TITLE
Add option to select from default backgrounds

### LINKED ISSUE #8: Provide an option to select a PNG from default images
Provide an option to choose from a set of default images (in /uploads folder) so that user can create badges without the need to upload images themselves. The user should be able to select one (multiple?) image from the set of images and upload a CSV to generate badges.

### PR DESCRIPTION
**Pull request by jajodiaraghav**
Fixes #8 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

#### Changes proposed in this pull request:

-
-
-

### CODE REVIEW

**Comment by S2606** [comment]
@jajodiaraghav Can you share the heroku link for viewing?

**Comment by jajodiaraghav** [comment]
this is what I get on clicking on "Deploy to heroku"
capture

**Comment by S2606** [comment]
@jajodiaraghav Maybe there's some issue regarding heroku on your local machine because this is your deployed branch
https://badgeyaytest.herokuapp.com/

File: app/templates/index.html [hunk 1]
```diff
@@ -26,10 +26,29 @@
 					<div class="text-center">OR</div>
 					<label for="textArea">Enter data manually</label>
 					<textarea class="form-control" id="textArea" name="csv"></textarea>
+
 					<label for="imageFile">[Optional] Upload a PNG file for badge background</label>
 					<input type="file" id="imageFile" name="image">
+					<div class="btn-group">
```
Comment (S2606) [inline app/templates/index.html]: @jajodiaraghav Add a heading regarding selecting default background.

**Pull request merged**